libs/libqmi/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libqmi
-PKG_VERSION:=1.34.0
+PKG_VERSION:=1.35.4-dev
```

Comment:
Why we should use unstable release?

Reply:
> Why we should use unstable release?

It's because the needed functionality ATM is present in the `-dev` version only:
https://gitlab.freedesktop.org/mobile-broadband/libqmi/-/commit/08ccdb85979c3ae054f59268273c28c7945967c4
![image](https://github.com/openwrt/packages/assets/233979/4c2d58cb-82d1-438b-869a-0e9987b35b4c)